cartesian2polar.py

```diff
@@ -0,0 +1,14 @@
+from math import acos, sqrt, pi
+
+# Read values for x and y
+x = float(input("x = "))
+y = float(input("y = "))
+
+# Convert polar coordinates to cartesian
+r = sqrt(x**2 + y**2)
+θ = acos(x/r)
+θ *= 1 if y >= 0 else -1
+
+# Print results
+print("r = " + str(r))
+print("θ = " + str(θ))

```

Comment:
**Suggestion:** It's a good practice to encapsulate the conversion logic into a function. This will make the code more modular, reusable, and testable.
```suggestion
def cartesian_to_polar(x, y):
    r = sqrt(x**2 + y**2)
    if x >= 0:
        θ = acos(x/r)
    else:
        θ = 2*pi - acos(x/r)
    return r, θ

x = float(input("x = "))
y = float(input("y = "))
r, θ = cartesian_to_polar(x, y)
print("r = " + str(r))
print("θ = " + str(θ))
```